local_planner/src/nodes/star_planner.h

```diff
@@ -106,7 +107,7 @@ class StarPlanner {
                      double height_change_cost_param_adapted,
                      double height_change_cost_param);
   void setPose(const geometry_msgs::PoseStamped& pose);
-  void setBoxSize(const Box& histogram_box);
+  void setBoxSize(const Box& histogram_box, const double ground_distance);
```

Comment:
Missing &.
Passing by const value makes no sense, const & or by value.

Reply:
For a double, just by value makes the most sense I think.